src/api/status/public/js/build-log/build-header.js

```diff
@@ -42,11 +42,20 @@ function renderBuildInfo({ isCurrent, githubData, startedDate, stoppedDate, code
 
   buildHeaderTitle.innerHTML = '';
 
+  if (startedDate !== null && stoppedDate === null) {
+    buildResult.innerHTML = 'motion_photos_on';
+    buildResult.classList.add('fa-spin', 'text-warning', 'fa-12x');
+  } else if (code === 0 && stoppedDate !== null) {
+    buildResult.innerHTML = 'check_circle';
+    buildResult.classList.add('text-success');
+  } else if (code !== 0 && stoppedDate !== 'null') {
+    buildResult.innerHTML = 'cancel';
+    buildResult.classList.add('text-danger');
+  }
+
```

Comment:
Icon without text can be ambiguous, so let's keep `buildResult` as text as before, then add another element for icons. 
Also, these should be moved into a function like the others below.

Reply:
Right now it kind of hard for me to test it out. I pinged Josue to trigger build manually but they seem not working and not receiving build information. I will try to work on it. Any guidance ?

Reply:
The terminal is showing build-log but the build-header is not showing what is being built?

Reply:
https://github.com/Seneca-CDOT/telescope/blob/dc273ca972bd1a8684042796a619121da9c9efa7/src/api/status/public/js/build-log/build-header.js#L62 It always went it to this line right away

Reply:
What is happening in https://github.com/Seneca-CDOT/telescope/blob/dc273ca972bd1a8684042796a619121da9c9efa7/src/api/status/public/js/build-log/check-for-build.js#L9 ?

Reply:
I think I pick it up from your PR #2653 . By the way how can you debug it manually without trigger any build ?

Reply:
No, I mean when `build` is null, what is being sent to you?  The line I gave you is where it picks the build.

You can't debug it without doing it for real, which is what I was saying on Slack yesterday about needing to find a way to do it..

Reply:
I pinged you on Slack